payjoin/src/lib.rs

```diff
@@ -30,6 +30,8 @@ pub mod send;
 #[cfg(feature = "v2")]
 pub(crate) mod hpke;
 #[cfg(feature = "v2")]
+pub use hpke::{HpkeKeyPair, HpkePublicKey};
```

Comment:
should be 
`pub use crate::hpke::{HpkeKeyPair, HpkePublicKey};`
to be compatible with msrv